lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -15424,4 +15424,22 @@ msgid "Filter created"
 msgstr "Filtre créé"
 
 msgid "Filter saved"
-msgstr "Filtre sauvegardé"
\ No newline at end of file
+msgstr "Filtre sauvegardé"
```

Comment:
Do you think `enregistré` is a better translation than `sauvegardé` ?

Reply:
I wouldn't say one is better than the other, but we should be consistent in its use.